students/barb_matthews/lesson3/list_lab.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+## Lesson 3.2 List Lab
+## By: B. Matthews
+## 9/15/2020
+
+## Add a fruit to the list
+fruits = ['Apples', 'Pears', 'Oranges', 'Peaches']
+yummyfruit = ['something']
```

Comment:
All variables in Python (and almost everything else except Classes) should by snake_case.  So, yummy_fruit